config/initializers/delayed_job.rb

```diff
@@ -6,8 +6,8 @@
 Delayed::Worker.delay_jobs = !Rails.env.test?
 Delayed::Worker.sleep_delay = (ENV['DELAYED_JOB_SLEEP_DELAY'].presence || 10).to_f
 
-# Delayed::Worker.logger = Logger.new(Rails.root.join('log', 'delayed_job.log'))
-# Delayed::Worker.logger.level = Logger::DEBUG
+Delayed::Worker.logger = Logger.new(STDOUT)
```

Comment:
Shouldn't that be `Rails.logger` which would also adapt the log level of the `Rails.logger`?